docs/content/guide/migration.ngdoc

```diff
@@ -43,7 +43,7 @@ services, directives, filters and helper functions. Existing applications can st
 benefits of such changes in `$compile`, `$parse`, `$animate`, `$animateCss`, `$sanitize`, `ngOptions`,
 `currencyFilter`, `numberFilter`, `copy()` (to name but a few) without any change in code.
 
-New features have been added to more than a dozen services, directives and filters across 7 modules.
+New features have been added to more than a dozen services, directives and filters across 8 modules.
 Among them, a few stand out:
```

Comment:
I didn't add the one-way bindings in the "highlights" (the list is getting too long for the migration guide).
I'm fine adding it though if someone feels it's worth highlighting (or maybe remove some other item).